openfl/component/collaborator/collaborator.py

```diff
@@ -14,6 +14,8 @@
 from openfl.protocols import utils
 from openfl.utilities import TensorKey
 
+import wandb
+LOG_WANDB = True
```

Comment:
Same comment as above. Should only import this if `wandb` is required in the plan